test/unit-test.js

```diff
@@ -34,10 +35,36 @@ describe('output-wrapper', function() {
 
   it(`should not allow other fs operations`, function() {
     expect(() => output.writevSync('test.md', 'test')).to.throw(
-      `Operation writevSync is not allowed to use. Allowed operations are readFileSync,existsSync,lstatSync,readdirSync,statSync,writeFileSync,appendFileSync,rmdirSync,mkdirSync`
+      /^Operation writevSync is not allowed to use. Allowed operations are readFileSync,existsSync,lstatSync,readdirSync,statSync,writeFileSync,appendFileSync,rmdirSync,mkdirSync,unlinkSync,symlinkOrCopySync$/
     );
   });
 
+  it('can remove folder recursively', function() {
+    output.mkdirSync('test');
+    expect(fs.existsSync(`${temp.name}/test`)).to.be.true;
+    output.writeFileSync('test.md', 'test');
+    output.rmdirSync('./', {
+      recursive: true
+    });
+    expect(fs.existsSync(`${temp.name}/test`)).to.be.false;
+  });
+
+  it('can remove folder non recursive', function() {
```

Comment:
Can we test that non recursive fails if it is used to delete a directory which contains content? And that it fails the same way the new version fails?

Reply:
https://github.com/SparshithNR/broccoli-output-wrapper/blob/61e46ea7c427dff35b1c5c43cf32408cef209cd3/test/unit-test.js#L55

I have that test added.